### DISCUSSION TITLE
发件权限不需要管理员审核,直接就通过审请了?

### DISCUSSION THREAD

**Discussion by 80imike** [dreamhunter2333/cloudflare_temp_email#297] [Q&A] [answered]
我看发件权限不需要管理员审核, 直接就通过审请了? 

是这样设计的吗？

这里一点申请权限就自动审核通过了 

<img width="1126" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/e622408c-a2fa-41c1-94ef-2f5815caa89d">

**Comment by dreamhunter2333** [comment]
> 我看发件权限不需要管理员审核, 直接就通过审请了? 
> 
> 
> 
> 是这样设计的吗？
> 
> 
> 
> 这里一点申请权限就自动审核通过了 
> 
> 
> 
> <img width="1126" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/e622408c-a2fa-41c1-94ef-2f5815caa89d">
> 
> 

可以配置默认数量，目前默认为1，config 中可以配置

**Comment by dreamhunter2333** [comment]
话说你的服务可以正常用么，cf d1 貌似挂了
https://www.cloudflarestatus.com/incidents/3xmdy3cdn85v?_ga=2.105231939.1710463395.1716950618-2028349030.1693759915

**Comment by 80imike** [comment, marked as answer]
> > 我看发件权限不需要管理员审核, 直接就通过审请了?
> > 是这样设计的吗？
> > 这里一点申请权限就自动审核通过了
> > <img alt="image" width="1126" src="https://private-user-images.githubusercontent.com/1665201/334634637-e622408c-a2fa-41c1-94ef-2f5815caa89d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTI1NDMsIm5iZiI6MTcxNjk1MjI0MywicGF0aCI6Ii8xNjY1MjAxLzMzNDYzNDYzNy1lNjIyNDA4Yy1hMmZhLTQxYzEtOTRlZi0yZjU4MTVjYWE4OWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMDMxMDQzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzIxMWRmM2JkNTAxZDBkNWI1OWFjZWI0NmFlOTA5NjcwZTc0YmExYTAzMzU4MzMzZDEwZDFlYTYyNTkyYzBhMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.-Yw6EOOM4Xvsdui35QFV14PDa0H-BhQ-u3V3lmCLVdU">
> 
> 可以配置默认数量，目前默认为1，config 中可以配置

我看了下 设成 0 就需要管理员后台审核了，原来是这样设计的

**Comment by 80imike** [comment]
> 话说你的服务可以正常用么，cf d1 貌似挂了 https://www.cloudflarestatus.com/incidents/3xmdy3cdn85v?_ga=2.105231939.1710463395.1716950618-2028349030.1693759915

我现在还可以用的

**Comment by 80imike** [comment]
<img width="1334" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/7ac8f26a-9141-4556-8708-0f3194945749">

大佬 帐号删除后，发件权限这里好像没有同步清理？ @dreamhunter2333

**Comment by dreamhunter2333** [comment]
> <img width="1334" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/7ac8f26a-9141-4556-8708-0f3194945749">
> 
> 
> 
> 大佬 帐号删除后，发件权限这里好像没有同步清理？ @dreamhunter2333 
> 
> 

这个就是昨天另一个 issue 的 bug，正在修复删除账号功能，是没有检查 kv 是否绑定导致的

**Comment by 80imike** [comment]
> > <img alt="image" width="1334" src="https://private-user-images.githubusercontent.com/1665201/334636691-7ac8f26a-9141-4556-8708-0f3194945749.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTM4NDAsIm5iZiI6MTcxNjk1MzU0MCwicGF0aCI6Ii8xNjY1MjAxLzMzNDYzNjY5MS03YWM4ZjI2YS05MTQxLTQ1NTYtODcwOC0wZjMxOTQ5NDU3NDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMDMzMjIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmQzMDU5YzNjOGFkYzUxYWYyZTgxYjY3MmI4NmZhYTk1NTEwOTE4ZTI3NmU2YzE1ZGIyODA4ODMzNzcwOTJmMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JbZmo9N3WFN48fHiGoNkBeSXHXhiAFcbd3yQE3w6eX0">
> > 大佬 帐号删除后，发件权限这里好像没有同步清理？ @dreamhunter2333
> 
> 这个就是昨天另一个 issue 的 bug，正在修复删除账号功能，是没有检查 kv 是否绑定导致的

好的，期待大佬早日修复

**Comment by dreamhunter2333** [comment]
> > > <img alt="image" width="1334" src="https://private-user-images.githubusercontent.com/1665201/334636691-7ac8f26a-9141-4556-8708-0f3194945749.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTM4NDAsIm5iZiI6MTcxNjk1MzU0MCwicGF0aCI6Ii8xNjY1MjAxLzMzNDYzNjY5MS03YWM4ZjI2YS05MTQxLTQ1NTYtODcwOC0wZjMxOTQ5NDU3NDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMDMzMjIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmQzMDU5YzNjOGFkYzUxYWYyZTgxYjY3MmI4NmZhYTk1NTEwOTE4ZTI3NmU2YzE1ZGIyODA4ODMzNzcwOTJmMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JbZmo9N3WFN48fHiGoNkBeSXHXhiAFcbd3yQE3w6eX0">
> > > 大佬 帐号删除后，发件权限这里好像没有同步清理？ @dreamhunter2333
> > 
> > 
> > 这个就是昨天另一个 issue 的 bug，正在修复删除账号功能，是没有检查 kv 是否绑定导致的
> 
> 好的，期待大佬早日修复

v0.4.6 已修复
https://github.com/dreamhunter2333/cloudflare_temp_email/releases/tag/v0.4.6

**Comment by 80imike** [comment]
> > > > <img alt="image" width="1334" src="https://private-user-images.githubusercontent.com/1665201/334636691-7ac8f26a-9141-4556-8708-0f3194945749.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTM4NDAsIm5iZiI6MTcxNjk1MzU0MCwicGF0aCI6Ii8xNjY1MjAxLzMzNDYzNjY5MS03YWM4ZjI2YS05MTQxLTQ1NTYtODcwOC0wZjMxOTQ5NDU3NDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMDMzMjIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmQzMDU5YzNjOGFkYzUxYWYyZTgxYjY3MmI4NmZhYTk1NTEwOTE4ZTI3NmU2YzE1ZGIyODA4ODMzNzcwOTJmMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JbZmo9N3WFN48fHiGoNkBeSXHXhiAFcbd3yQE3w6eX0">
> > > > 大佬 帐号删除后，发件权限这里好像没有同步清理？ @dreamhunter2333
> > > 
> > > 
> > > 这个就是昨天另一个 issue 的 bug，正在修复删除账号功能，是没有检查 kv 是否绑定导致的
> > 
> > 
> > 好的，期待大佬早日修复
> 
> v0.4.6 已修复 https://github.com/dreamhunter2333/cloudflare_temp_email/releases/tag/v0.4.6

已更新，亲测好用 赞
